components/add.js

```diff
@@ -216,10 +217,9 @@ export const AddInstancePage = ({ collection }) => {
               })
             : [
                 {
-                  description:
-                    "Error saving new item, ensure the fields are filled out correctly",
-                  keys: "Generic Error",
-                  key: "Generic ErrorError saving new item, ensure the fields are filled out correctly",
+                  description: defaultDescription,
+                  keys: defaultKeys,
+                  key: `${defaultKeys}${defaultDescription}`,
```

Comment:
Actually, I still don’t understand this key. Is it not for display?

Reply:
it's not, no